tools/buildtools/ambertools/acpype.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="acpype" name="Generate MD topologies for small molecules" version="0.1.0">
```

Comment:
track conda package version rather? I think we have been generally tracking against the conda package version and then would add minor codes if the tool xml changes a great deal.

Reply:
I'm not a huge fan of `2019.03.22.14.36` as a version name, but you're right, it's better to be consistent.